src/components/Navbar/Components/ProjectItemCard.tsx

```diff
@@ -20,7 +22,9 @@ const ProjectItemCard = ({ name, color }: ProjectItemCardProps) => {
     >
       <ColorDotIcon color={color} />
       <Text fontSize="16px">{name} </Text>
-      <Flex ml="auto">...</Flex>
+      <MyTooltip label={"More projects actions"}>
```

Comment:
If we use ""  dont need to have {}
```suggestion
      <MyTooltip label="More projects actions">
```